### PR TITLE
Make nav and search links refer to V1 of the site

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What is this PR trying to achieve?
Making links reference the V1 website where we do not have content on this site.
It will give us a better idea of how the launch is going to work now that we've decided what it looks like.

### CODE REVIEW

File: server/views/layout/default.njk [hunk 1]
```diff
@@ -16,10 +16,23 @@
...
+        title: "Visit us"
+      },
+      {
+        href: "https://wellcomecollection.org/whats-on",
+        title: "What's on"
+      },
+      {
+        href: "/explore",
+        title: "Explore",
+        isCurrent: pageConfig.inSection === 'explore'
```
Comment (alexwlchan) [inline server/views/layout/default.njk]: Why does this navlink get to keep its `isCurrent` attribute when the others don’t?

File: server/views/components/header/index.njk [hunk 2]
```diff
@@ -30,11 +30,13 @@
...
       <div id="header-search" class="header__search">
         <form class="header__form"
-          action="search"
+          action="https://wellcomecollection.org/search"
           method="post">
           <div class="header__input-wrap js-show-hide-drawer">
             <label class="header__label" for="header-input">Search</label>
-            <input id="header-input" class="header__input js-header-input" placeholder="Search" />
+            <input id="header-input" class="header__input js-header-input" placeholder="Search" name="search" />
+            {# This is here as we're sending people back to V1, abd drupal requires it #}
```
Comment (alexwlchan) [inline server/views/components/header/index.njk]: /s/abd/and

Comment (jamesgorrie) [inline server/views/components/header/index.njk]: 👀

File: server/views/layout/default.njk (hunk 1 above)
Comment (jamesgorrie) [inline server/views/layout/default.njk]: The templates unfortunately don't have a method definition or a defined list of arguments (i.e. We can't say what arguments are required) - so not sending this over get's evaluated as `falsey`.

If we could specify a template function, I would have:
```
class NavLink(title: String, href: URL, isCurrent: Boolean = false)
def navigationTemplate(navLinks: List[NavLink])
```

I am looking at getting at least the server code typed (one of the reasons I am [removing WebPack](https://github.com/wellcometrust/wellcomecollection.org/pull/299)) which should help with some of this.

I could have left the code `isCurrent: pageConfig.inSection === 'what-we-do'` etc in - but felt it was misleading as that content doesn't exist.

Comment (alexwlchan) [inline server/views/layout/default.njk]: Ah, I see – I hadn’t noticed that this was the only navlink pointing to a page on the new site. Makes sense, thanks.

**Pull request merged**